vignettes/packrat.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
-title: "packrat vs renv"
+title: "Packrat vs. renv"
```

Comment:
FWIW I think it's ok to start a sentence/heading with a lower case word if that's a proper noun.

Reply:
Makes sense. I was also surprised to learn that "vs" versus "vs." is apparently a British vs. American English thing too. 🙃